starter/critter/src/main/resources/application.properties

```diff
@@ -4,7 +4,14 @@ server.port=8082
 # Data Source Setup
 # Note: Users must create a mysql db that they will reference here, create a user that has access to it on localhost
 #       and provide the credentials below to access it
```

Comment:
Providing these properties for 'free' in the starter code is kind of giving away part of the solution here. I don't really care what you guys do with this at this point, but they were excluded intentionally.